### ISSUE TITLE
organizer is a Person with a working url

### ISSUE THREAD

**Issue by nickevansuk** [openactive/activation#211]
## Issue
- `organizer` must be of type `Person` for individual instructors (e.g `Jane Smith`) and type `Organization` for a group (e.g `Active Seniors`). If this distinction does not exist within Classfinder's data entry pages, it should be added.
  - Note that having checked with @jadecation for now all organizers will be of type `Person`, and allowing instructors to specify this distinction to include the `Organization` if they are an organization has been added to the development list 
- `url` must link to instructor page (e.g. https://portal.emduk.org/find-an-instructor/197209)
- must include `identifier` with the ID of the instructor

## Example

Not like this:

```javascript
"organizer": {
  "name": "Jane Smith",
  "type": "Organization"
}
```

Like this:

```javascript
"organizer": {
  "type": "Person",
  "identifier": 197209,
  "name": "Jane Smith",
  "url": "https://portal.emduk.org/find-an-instructor/197209"
},
```

**Comment by vakeesan26** [comment]
is that leader and organizer properties going to be same ?

**Comment by nickevansuk** [comment]
Yes, except leader is an array with one item, and organizer is just the item (no array)
